cpp/src/strings/convert/convert_integers.cu

```diff
@@ -66,27 +165,28 @@ struct string_to_integer_fn {
 struct dispatch_to_integers_fn {
   template <typename IntegerType, std::enable_if_t<std::is_integral<IntegerType>::value>* = nullptr>
   void operator()(column_device_view const& strings_column,
-                  mutable_column_view& output_column,
+                  mutable_column_view const& output_column,
```

Comment:
```suggestion
                  mutable_column_view& output_column,
```
This seems wrong. The `output_column` is being modified and therefore should not be `const`.

Reply:
I thought about that too, but it compiles and works fine, thus I'm not sure if this is a compiler bug, or it actually "not wrong"???

Reply:
Either way, it communicates the wrong intent of this object.

Reply:
I see. I'll change it to normal ref.